src/app/views/detail-idea/detail-idea.component.ts

```diff
@@ -47,6 +48,18 @@ export class DetailIdeaComponent {
     }, 1000);
   }
 
+  onSubmitComment(){
+    console.log(`Comentario: ${this.commentContent}`);
+    // TODO: Save comment in DB
+
+    // Clear textarea content
+    this.commentContent = "";
+  }
+
+  isCommentWritten(){
+    return this.commentContent !== "";
```

Comment:
Add here a trim to avoid possible comments only with a blank space, `this.commentContent.trim() !== ""`